tests/parsers/chemometec_nucleoview/chemometec_nucleoview_parser_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from allotropy.parser_factory import Vendor
+from tests.parsers.test_utils import from_file, validate_schema
+
+output_files = (
+    "chemometec_nucleoview_example01",
+    "chemometec_nucleoview_example02",
+    "chemometec_nucleoview_example03",
```

Comment:
could you please put the .csv extensions in here?

Reply:
done